tests/utils/common/common.py

```diff
@@ -32,6 +28,79 @@
 import traceback
 
 
+class Result:
+    def __init__(self, stdout, stderr, exited):
+        self.stdout = stdout
+        self.stderr = stderr
+        self.exited = exited
+        self.return_code = exited
```

Comment:
TIL! I have a whole bunch of things to modernize then, once I can actually run them! 😂 One more reason to upgrade!

Reply:
Turns out using dataclass won't work, because there is one small exception in the code: Look at the `exited` and `return_code`, they are both using the same input. I know this is kind of redundant, but to stay compatible with Fabric I'd like to keep both.